lib/ferver/controller.rb

```diff
@@ -58,6 +52,12 @@ def valid_file_request?
       file_id_request.valid?
     end
 
+    def find_file
+      ferver_list.file_by_id(file_id_request.value)
+    rescue IndexError
+      halt 404, 'File requested not found.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.